src/Config/Config.hpp

```diff
@@ -67,14 +67,14 @@ namespace nexusZiaApi {
 	   * @param key
 	   * @return const IConfigKey &
 	   */
-	  virtual const IConfigKey & getKeyOne(const std::string & key) = 0;
+	  virtual const IConfigKey & getKeyOne(const std::string & key) const = 0;
 
 	  /**
 	   * Get list of key in config of module
 	   * @param key
 	   * @return std::list<IConfigkey>
```

Comment:
Please update the documentation too (vector instead of list)

Reply:
done

Reply:
re-done